tencentcloud/resource_tc_alb_server_attachment.go

```
@@ -0,0 +1,292 @@
+					req.Backends = append(req.Backends, lbNewBackend(inst["instance_id"], inst["port"], inst["weight"]))
+				}
+				resp, err := client.DeregisterInstancesFromForwardLB(req)
+				if err != nil {
+					return resource.NonRetryableError(err)
+				}
+				// 9003 error code backend server with port not exist, not ncesssary to remove
+				if !strings.HasPrefix(*resp.Message, "(9003)") {
+					var wait = func() {
+						time.Sleep(2 * time.Second)
+					}
+
+					err = lbRequestStatusCheck(m, resp.RequestId, wait)
+					if err != nil {
+						return resource.NonRetryableError(err)
+					}
+				}
+				return nil
+			})
+		}
+		if len(add) > 0 {
+			client := m.(*TencentCloudClient).lbConn
+			return resource.Retry(5*time.Minute, func() *resource.RetryError {
+
+				req := lb.NewRegisterInstancesWithForwardLBSeventhListenerRequest()
+				req.LoadBalancerId = common.StringPtr(d.Get("loadbalancer_id").(string))
+				req.ListenerId = common.StringPtr(d.Get("listener_id").(string))
+
+				if loc, ok := d.GetOk("location_id"); ok {
+					req.LocationIds = []*string{}
+					l := loc.(string)
+					req.LocationIds = append(req.LocationIds, &l)
+				}
+				for _, inst_ := range add {
+					inst := inst_.(map[string]interface{})
+					req.Backends = append(req.Backends, lbNewBackend(inst["instance_id"], inst["port"], inst["weight"]))
+				}
+				resp, err := client.RegisterInstancesWithForwardLBSeventhListener(req)
+				if err != nil {
+					return resource.NonRetryableError(err)
+				}
+
+				var wait = func() {
+					time.Sleep(2 * time.Second)
+				}
+
+				err = lbRequestStatusCheck(m, resp.RequestId, wait)
+				if err != nil {
+					return resource.NonRetryableError(err)
+				}
+				return nil
+			})
+		}
+	}
+	return nil
+}
+
+func resourceTencentCloudAlbServerAttachmentRead(d *schema.ResourceData, m interface{}) error {
+	client := m.(*TencentCloudClient).lbConn
+	return resource.Retry(5*time.Minute, func() *resource.RetryError {
```

read doesn't need to retry, if read fail, let it fail